### ISSUE TITLE
Journalist replies only support ASCII characters

### ISSUE THREAD

**Issue by garrettr** [freedomofpress/securedrop#1343]
This is an oversight/classic Python 2.x problem. When a journalist submits a reply to a source, and includes non-ASCII characters in the reply, it causes an error:

```
ERROR:journalist:Exception on /reply [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/var/www/securedrop/journalist.py", line 86, in wrapper
    return func(*args, **kwargs)
  File "/var/www/securedrop/journalist.py", line 448, in reply
    output=store.path(g.sid, filename))
  File "/var/www/securedrop/crypto_util.py", line 161, in encrypt
    armor=False)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/gnupg.py", line 972, in encrypt
    stream = _make_binary_stream(data, self._encoding)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/_util.py", line 547, in _make_binary_stream
    thing = thing.encode(encoding)
UnicodeEncodeError: 'ascii' codec can't encode character u'\\u2019' in position 8: ordinal not in range(128)
```

**Comment by mdrose** [comment]
@garrettr I just tried this and was unable to reproduce it on either develop or 0.3.8. I put some Chinese and Russian text in a journalist reply and it sent with no errors. The message appeared normally on the source side, as well.

**Comment by garrettr** [comment]
@mdrose Huh, that's interesting. We got the bug report from the private support tracker that we use with SecureDrop administrators. I haven't had a chance to try to reproduce yet but will do today or tomorrow.

**Comment by fowlslegs** [comment]
Resolved by #1370.
